sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -981,22 +981,114 @@ func marshalWindowingStrategy(c *CoderMarshaller, w *window.WindowingStrategy) (
 	} else {
 		mergeStat = pipepb.MergeStatus_NON_MERGING
 	}
+	trigger := makeTrigger(w.Trigger)
+	accMode := makeAccumulationMode(w.AccumulationMode)
```

Comment:
We can inline these calls to the proto field assignments instead of assigning to variables first.